cli/command/stack/config.go

```diff
@@ -0,0 +1,62 @@
+package stack
+
+import (
+	"fmt"
+
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/command"
+	"github.com/docker/cli/cli/command/stack/loader"
+	"github.com/docker/cli/cli/command/stack/options"
+	composeLoader "github.com/docker/cli/cli/compose/loader"
+	composetypes "github.com/docker/cli/cli/compose/types"
+	"github.com/spf13/cobra"
+	yaml "gopkg.in/yaml.v2"
+)
+
+func newConfigCommand(dockerCli command.Cli) *cobra.Command {
+	var opts options.Config
+
+	cmd := &cobra.Command{
+		Use:     "config [OPTIONS]",
+		Aliases: []string{"cfg"},
```

Comment:
Can we remove the alias for now? I think we should start smaller and if we have some feedback about it then we will re-add that alias in a follow-up 👍

Reply:
Removed alias.